docs/application/dotnet/samples/messaging/overview.md

```diff
@@ -38,3 +38,18 @@ for TD:
 
 <div class="tabcontent" id="TV">
 </div>
+
+<!--
+For MD:
+-->
+
+<link href="../css/dotnet-samples.css" ref="stylesheet">
```

Comment:
In other sample overviews, "../js/dotnet-samples.js" is used to apply script.
However only in this file "../css/dotnet-samples.css" file is used.
Is there any difference in this file?

Reply:
done.